docs/linking/README.md

```diff
@@ -162,6 +176,10 @@ should prompt the user for this secret and then provide that back to the DFSP.
 
 ![Authentication (OTP)](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/mojaloop/pisp/master/docs/linking/3-authentication-otp.puml)
 
+#### DFSPAuthorizeSimulator
+DFSPAuthorizeSimulator is api service for demo purposes. For the `OTP flow` it exposes `/sendOTP` API endpoint which allows to simulate sending SMS token to the User's device.
+
+It is implemented as extension for TestingToolkit service which allows to define testing scenarios as TTK rules driven by `username` parameter.
```

Comment:
I'm not quite sure I see how this works. Will there be a way for the PISP to configure the username? Or will it just be the DFSP?

Reply:
@lewisdaly `sendOTP` is API used by DFSP, we added it so we can simulate sending SMS in the OTP flow, we plan to first implement it in TTK, later we can use AWS SNS to send real SMS - or any other service if needed

Reply:
@lewisdaly with @sridharvoruganti we decided that we will implement that DFSP needs to store consentRequest details somewhere - actions SIM-1..3 - here in DFSPAuthorizeSimulator, so later the DFSPLoginApp can use `GET /consentRequests/<id>` to retrieve the details, including the list of accounts related to this consent request grant SIM-12..15

Reply:
story to implement generic store in TTK: https://app.zenhub.com/workspaces/mojaloop-project-59edee71d1407922110cf083/issues/mojaloop/project/1986